### ISSUE TITLE
[Discussion] IDE Code Folding

### ISSUE THREAD

**Issue by damianday** [beefytech/Beef#363]
Please give the issue a thumbs up if you would like the IDE to support such a feature, such as regions, method folding, etc.

**Comment by Shadowblitz16** [comment]
I thin code folding should be like so..

```
void Method() { ... }
```

and not ```
void Method() {
}
```

I also think that there should be code folding options in the editor...
- fold on load
- unfold on load
- keep on load

**Comment by bfiete** [comment]
Implemented at 90735e3bf88d91145d6c1cb500d42211b55fce48
